app/assets/javascripts/oxalis/model/sagas/save_saga.js

```diff
@@ -182,14 +182,10 @@ export function* sendRequestToServer(
     yield* call(toggleErrorHighlighting, false);
   } catch (error) {
     yield* call(toggleErrorHighlighting, true);
-    if (error.status >= 400 && error.status < 500) {
+    if (error.status === 409) {
       // HTTP Code 409 'conflict' for dirty state
       window.onbeforeunload = null;
-      if (error.status === 409) {
-        yield* call(alert, messages["save.failed_simultaneous_tracing"]);
-      } else {
-        yield* call(alert, messages["save.failed_client_error"]);
```

Comment:
Maybe we should show a toast in this case, anyway? In case funny stuff happens, the user will be able to include these notifications in their bug reports. The message could be something like `An unknown error occurred (status code: ...). Saving will be retried in a few seconds.`

Reply:
The `toggleErrorHighlighting` function (some lines below) does take care of showing a toast in that case, it does not contain the status code, though.

Reply:
Ah, nice, then nevermind :)